wordpress-mobile/WordPressAuthenticator-iOS
# UL&S: Display an instruction label on the tableview

Ref #283 

This PR adds the reusable `InstructionTableViewCell` and nib file. The `UIView+Helpers` extension adds a convenience method for loading nib files. The `SiteAddressViewController` has been updated to load and configure the `InstructionTableViewCell` for viewing the changes. 

A table section / row enum for tableView configurations are coming next.

### To test
1. Check out this branch
2. `bundle exec pod install` to ensure there are no `WordPressKit` error messages in the terminal
3. Build and run
4. Check out the test PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/14333
